src/common/vocabulary/actions.js

```diff
@@ -20,6 +21,31 @@ export default function actions(state) {
   };
 
   return {
+    //become a card(becomeACopy takes a playedObject, this takes a card)
+    become: function (sources, card) {
+      //console.log(card);
+      iterateOver(sources)(source=>{
+        Object.assign(source,{
+          card: cloneDeep(card),
+          stats: cloneDeep(card.stats),
+          abilities: [],
+          triggers: []
+        });
+      });
+
+      //set triggers
+      if (card.abilities && card.abilities.length > 0) {
+        card.abilities.forEach((cmd, idx) => {
+          const cmdText = splitSentences(card.text)[idx];
+          state.currentCmdText = cmdText.includes('"') ? cmdText.split('"')[1].replace(/"/g, '') : cmdText;
+
+          iterateOver(sources)(source=>{executeCmd(state, cmd, source);});//works for multitarget? who knows!
```

Comment:
So does it work?

Reply:
i think so? I still don't completely understand the client, so the code might be broken in some hidden way somewhere, but I don't know what to look/test for.